core/tests/puppeteer-acceptance-tests/specs/logged-in-user/manage-goals-progress-and-lessons-from-learner-dashboard.spec.ts

```diff
@@ -100,7 +100,7 @@ describe('Logged-in User', function () {
         'Congratulations for completing this lesson!'
       );
 
-      await loggedInUser.navigateToLearnerDashboard();
+      await loggedInUser.navigateToLearnerDashboardUsingProfileButton();
```

Comment:
Suggest "user dropdown" instead of "profile button", ditto below

Reply:
Done!